backend/src/chats/message/entities/message.entity.ts

```diff
@@ -0,0 +1,35 @@
+import {
+	Column,
+	CreateDateColumn,
+	Entity,
+	JoinColumn,
+	ManyToOne,
+	OneToOne,
+	PrimaryGeneratedColumn,
+	UpdateDateColumn,
+} from 'typeorm';
+import { Chat } from '../../chat/entities/chat.entity';
+import { User } from '../../../users/user/entities/user.entity';
+
+@Entity('messages')
+export class Message {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@CreateDateColumn()
+	created_at: Date;
+
+	@UpdateDateColumn()
+	updated_at: Date;
+
+	@OneToOne((type) => User, (user) => user.id)
+	@JoinColumn({ name: 'sender_id' })
+	sender_id: User;
+
+	@ManyToOne((type) => Chat, (chat) => chat.id)
```

Comment:
The relationships are uni-directional at this point right? Do we want to have them bi-directional? So the messages aren't accessible via the chat table?
So calling up the chat doesn't display the messages...

Reply:
I created an issue to solve this tomorrow!